charts/newrelic-pixie/README.md

```diff
@@ -0,0 +1,71 @@
+# newrelic-pixie (pre-release/early access program)
+
+**This chart is a pre-release and installs software under an early access program.**
+
+## Chart Details
+
+This chart will deploy the New Relic Pixie Integration.
+
+IMPORTANT: make sure you deploy this chart in the same namespace as Pixie. 
+It needs to access the cluster id inside the Pixie secrets. 
+By default, Pixie is installed in the `pl` namespace.
+
+## Configuration
+
+| Parameter                     | Description                                                  | Default                    |
+| ----------------------------- | ------------------------------------------------------------ | -------------------------- |
+| `global.cluster` - `cluster`  | The cluster name for the Kubernetes cluster. Required.       |                            |
+| `global.licenseKey` - `licenseKey` | The New Relic license key (stored in a secret). Required.    |                            |
+| `global.nrStaging` - `nrStaging` | Send data to staging (requires a staging license key). | false |
+| `apiKey`                      | The Pixie API key (stored in a secret). Required.            |                            |
+| `verbose`                     | Whether the integration should run in verbose mode or not.   | false                      |
+| `global.customSecretName` - `customSecretName` | Name of an existing Secret object, not created by this chart, where the New Relic license is stored                                                                                                                                                                         |                                 |
+| `global.customSecretLicenseKey` - `customSecretLicenseKey` | Key in the existing Secret object, indicated by `customSecretName`, where the New Relic license key is stored.                                                                                                                                                             |                                 |
+| `customSecretApiKeyName` | Name of an existing Secret object, not created by this chart, where the Pixie API key is stored.    |                                 |
+| `customSecretApiKeyKey` | Key in the existing Secret object, indicated by `customSecretApiKeyName`, where the Pixie API key is stored.   |                                 |
+
+## Example
+
+Make sure you have [added the New Relic chart repository.](../../README.md#installing-charts)
+
+Then, to install this chart, run the following command:
+
+```sh
+helm install newrelic/newrelic-pixie \
+  --set cluster=<Kubernetes cluster name> \
+  --set licenseKey=<Your New Relic license key> \
+  --set apiKey=<Your Pixie API key> \ 
+  --namespace pl \
+  --generate-name
```

Comment:
Isn't better to specify an example name instead of using `--generate-name`?

Reply:
No. In fact, it's dangerous for us to specify a fixed name here. Imagine the scenario where a cluster is used by two different teams, on different namespaces. One team installs this chart and configures it for them. Then our instructions tell the other team to also install it with the same name, completing breaking the chart for the previous team.

If the default behavior is generate name, this problem doesn't happen.